music/views.py

```diff
@@ -418,6 +419,59 @@ def get(self, request):
                     "lyrics_list"    : lyrics_list
                 }, status = 200)
 
+class DomesticRankingView(View):
+    def get(self, request):
+        limit   = request.GET.get('limit', 100)
+        COUNTRY = request.GET.get('COUNTRY', '한국')
 
+        albums = (
+            ArtistAlbum
+            .objects
+            .filter(artist__nationality__icontains=COUNTRY)
+            .order_by('-album__like_count')[:limit]
+        )
+        
+        domestic_like_album_list = [{
+            'album_id'             : album.album.id,
+            'album_name'           : album.album.name,
+            'album_image'          : album.album.image,
+            'album_artist_name'    : album.artist.name
+        } for album in albums ]
+        return JsonResponse({"domestic_like_album_list": domestic_like_album_list}, status = 200)
+
+class ForeignRankingView(View):
+    def get(self, request):
+        limit   = request.GET.get('limit', 100)
+        COUNTRY = request.GET.get('COUNTRY', '한국')
```

Comment:
변수 소문자로 변경 부탁 드립니다. 보통 전역변수나 기타 상수 제외하고는 대문자로 선언하지 않습니다.
```suggestion
        country = request.GET.get('country', '한국')
```

Reply:
아 넵, 변경하겠습니다~!